docs/user-guide/expressions/column_selections.md

```diff
@@ -0,0 +1,116 @@
+
+# Column Selections
+
+Let's create a dataset to use in this section:
+
+{{code_block('user-guide/expressions/column_selections','selectors_df',['DataFrame'])}}
+
+```python exec="on" result="text" session="user-guide/column_selections"
+--8<-- "python/user-guide/expressions/column_selections.py:setup"
+--8<-- "python/user-guide/expressions/column_selections.py:selectors_df"
+```
+
+## Expression Expansion
+
+As we've seen in the previous section, we can select specific columns using the `pl.col` method. It can also select multiple columns - both as a means of convenience, and to *expand* the expression.
+
+This kind of convenience feature isn't just decorative or syntactic sugar. It allows for a very powerful application of [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principles in your code.
```

Comment:
Can you add what the expansion does? Maybe add a few code snippets that show how the column selection and some compute result into multiple evaluated expressions. It should be very clear that expansion isn't just selection of columns, but select + compute combined.

Reply:
I'll add what expansion by itself does.

I've added code examples with for all kinds of expansion - is that not what you're looking for?